

def write_errors_in_file ():
    pass